server/methods/initiative.js

```diff
@@ -1,7 +1,32 @@
+/*
+ * Global function to help reduce boilerplate code for Meteor
+ * checks on the initiative object
+ * @param initiative
+ */
+checkInitiative = function checkInitiative(initiative) {
+  check (initiative, {
+    _id: String,
+    active: Boolean,
+    category: String,
+    categorySlug: String,
+    createdAt: Number,
+    createdBy: String,
+    imageUrl: String,
+    slug: String,
+    summary: String,
+    title: String,
+    comments: Match.Optional ([String]),
```

Comment:
`Match` is a meteor global. It needs to be defined with the rest of them in .eslintrc. Also, no space after `Match.Optional`